src/ovp_pipeline/absorb_router.py

```
@@ -0,0 +1,453 @@
+``docs/canonical-write-ownership.md`` for how this fits the M18
+trust-aware-compiler hardening.
+"""
+
+from __future__ import annotations
+
+import json
+import logging
+import sqlite3
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any, Iterable
```

Ruff (0.15.12)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)